be/src/Project/Web/Buyer/Buyer.Web/Areas/Products/Controllers/OutletController.cs

```diff
@@ -32,7 +36,8 @@ public async Task<IActionResult> Index()
                 Language = CultureInfo.CurrentUICulture.Name,
                 IsAuthenticated = this.User.Identity.IsAuthenticated,
                 Name = this.User.Identity.Name,
-                BasketId = string.IsNullOrWhiteSpace(this.Request.Cookies[BasketConstants.BasketCookieName]) ? null : Guid.Parse(this.Request.Cookies[BasketConstants.BasketCookieName])
+                BasketId = string.IsNullOrWhiteSpace(this.Request.Cookies[BasketConstants.BasketCookieName]) ? null : Guid.Parse(this.Request.Cookies[BasketConstants.BasketCookieName]),
+                SellerId = GuidHelper.ParseNullable((User.Identity as ClaimsIdentity).Claims.FirstOrDefault(x => x.Type == AccountConstants.Claims.OrganisationIdClaim)?.Value)
```

Comment:
I need the SellerId to get data about the client's transport, zone, and campaign to get the completion date from the API.

Reply:
I understand but SellerId is equal organisation id not client id :D

maybe it would be worth passing this information in the session? or creating an endpoint that will collect information about the client via email